### PR TITLE
Sonar cloud coverage

### PR DESCRIPTION
**Pull request by abdrabrcom**

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+sonar.projectBaseDir=che-che4z-lsp-for-cobol/com.ca.lsp.cobol
+sonar.modules=lsp-core-cobol-parser,lsp-core-domain,lsp-service-cobol
+sonar.sources=lsp-core-cobol-parser/src/main,lsp-core-domain/src/main,lsp-service-cobol/src/main
+sonar.tests=lsp-core-cobol-parser/src/test,lsp-core-domain/src/test,lsp-service-cobol/src/test
+sonar.java.binaries=lsp-service-cobol/target
+sonar.coverage.jacoco.xmlReportPaths=lsp-service-cobol/target/coverage-reports/jacoco-unit.exec
+sonar.language=java
+sonar.coverage.exclusions=lsp-core-cobol-parser/src/main/antlr4,lsp-core-cobol-parser/src/main/resources,lsp-core-domain/src/main/resources,lsp-service-cobol/src/main/resources
+
+#sonar.organization=eclipse
```
Comment (zacanbrcom) [inline sonar-project.properties]: remove commented code

File: sonar-project.properties [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+#sonar.projectBaseDir=com.ca.lsp.cobol
+#sonar.modules=lsp-core-cobol-parser,lsp-service-cobol,lsp-core-domain
+#sonar.sources=src/main/java
+#sonar.tests=src/test/java
+#sonar.java.binaries=target/classes
```
Comment (zacanbrcom) [inline sonar-project.properties]: remove commented code

File: com.ca.lsp.cobol/pom.xml [hunk 3]
```diff
@@ -28,7 +28,7 @@
     <lsp4j.version>0.5.0</lsp4j.version>
     <antlr.version>4.7.1</antlr.version>
   </properties>
-
+
```
Comment (zacanbrcom) [inline com.ca.lsp.cobol/pom.xml]: remove this extra space

Comment (abdrabrcom) [reply com.ca.lsp.cobol/pom.xml]: removed

File: sonar-project.properties (hunk 2 above)
Comment (abdrabrcom) [reply sonar-project.properties]: removed

**Comment by zacanbrcom** [comment]
Closed in order to test old sonarcloud configuration

**Pull request closed without merging**